docs/docs/SUMMARY.md

```diff
@@ -37,7 +37,7 @@
     - [Logging](getting-started/logging.md)
     - [Config Management](getting-started/config/index.md)
 - [Kafka](kafka/index.md)
-    - [Subscription](kafka/Subscriber/index.md)
+    - [Subscribing](kafka/Subscriber/index.md)
```

Comment:
We should use the same section naming at all *RMQ*, *Kafka*, *NATS* and *Tutorial* blocks

Reply:
Someone messed it up - https://github.com/airtai/faststream/blame/0.2.0-NATS/docs/docs/summary_template.txt#L40. Let me fix it.

Reply:
Done.